src/entity/CommonGame.java

```diff
@@ -95,6 +95,9 @@ public Round getCurrentRound() {
     @Override
     public boolean isGameOver() { return allRounds.size() >= maxRounds || currentLives <= 0; }
 
+    @Override
+    public boolean isLoadable() { return finishedAt == null; }
```

Comment:
Yeah that's a good point and something I was contemplating as well. The problem I had with using isGameOver is that its use would currently lead to unexpected results. For example, suppose the user is on round 10/10 and they go back to the menu without finishing the round. In the current implementation, isGameOver would return true and thus the game wouldn't be loadable, although we know that it should be.

Saying that, I'm starting to think this says more about the incorrectness of isGameOver rather than advocating for the need of isLoadable. Perhaps we should edit CommonGame's isGameOver method to be more accurate and then substitute that in the use case for isLoadable. What do you think about this?

Reply:
Yes, lets add that as an issue and assign it. Maybe for the sake of this PR it is fine for now and once isGameOver() has been looked at and adjusted accordingly we can replace the isLoadable function which is only used in the DAO at the moment so it will be a very simple fix.

Reply:
Since I know how to fix it and it is applicable to these use cases, I should be able to handle that in this PR if that's alright with you!

Reply:
Oh 100%! I just suggested that in case you didn't want to do it! But ya for sure that sounds great